libraries/Microsoft.Bot.Builder.Ai.LUIS/Microsoft.Bot.Builder.Ai.LUIS.csproj

```diff
@@ -18,7 +18,6 @@
 	<PackageId>Microsoft.Bot.Builder.Ai.LUIS</PackageId>
     <Description>LUIS Middleware and Recognizers for the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using LUIS.</Summary>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Yes we want to remove from here, but we really need to _move_ to the conditional property group for `Release`.